src/util/hass-util.html

```diff
@@ -433,6 +432,32 @@
     if (stateObj.attributes.unit_of_measurement) {
       stateObj._stateDisplay += ' ' + stateObj.attributes.unit_of_measurement;
     }
+    switch (stateObj.attributes.device_class) {
```

Comment:
Device classes are only unique per domain so we should check for domain here too.